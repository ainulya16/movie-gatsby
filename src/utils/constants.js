export const theme = {
  color: {
    w: '#FFFFFF',
    k: '#000000',
    p1: '#D31145',
    p2: '#596C80',
    p3: '#554344',
    p4: '#363E3F',
    s1: '#E86487',
    s2: '#FBBF56',
    s3: '#97CB5D',
    s4: '#46D3BD',
    s5: '#22A8DA',
    s6: '#9962D2',
    t2: '#B8123E',
    b2: '#FAF9F3',
    b3: '#F0EEE1',
    b5: '#B3B1A8',
    u2: '#ce3c3a',
    u3: '#405B70',
    u4: '#769FAB',
    u5: '#c1272d',
    w9: '335px',
    b9: '#EEF0F2',
    whatsapp: '#25d366',
  },
};

export default null;
