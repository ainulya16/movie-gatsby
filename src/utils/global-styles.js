import { createGlobalStyle } from 'styled-components';
import 'slick-carousel/slick/slick.css';
import 'slick-carousel/slick/slick-theme.css';
import 'antd/dist/antd.css';
// import '../assets/css/slick.min.css';

const GlobalStyle = createGlobalStyle`
  body {
    font-family: 'AIAMedium';
  }
  @media (max-width: 575px) {
    .wrap_center .ant-form-item-label {
      text-align: center !important;
    }
  }
`;

export default GlobalStyle;
