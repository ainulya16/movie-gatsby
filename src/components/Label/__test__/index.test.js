import React from 'react';
import { shallow } from 'enzyme';

// import jest-styled-components for snapshot styled components
import 'jest-styled-components';
import renderer from 'react-test-renderer';

// import components to be tested
import Component from '..';
import { theme } from '../../../utils/constants';

describe('Label Component', () => {
  it('should be exist"', () => {
    const component = shallow(<Component theme={theme}>Label</Component>);
    expect(component.exists()).toBe(true);
  });
  it('should be match snapshot', () => {
    const tree = renderer
      .create(<Component theme={theme}>Label</Component>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should be red as error message style', () => {
    const tree = renderer
      .create(
        <Component theme={theme} error>
          Label
        </Component>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
