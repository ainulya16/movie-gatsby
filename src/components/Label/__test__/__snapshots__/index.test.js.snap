// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component should be match snapshot 1`] = `
.c0.c0 {
  color: #554344;
  font-family: 'AIABody',Arial,'Helvetica Neue',Helvetica,sans-serif;
  font-weight: bold;
  font-size: 15px;
  line-height: 20px;
}

<label
  className="c0"
>
  Label
</label>
`;

exports[`Label Component should be red as error message style 1`] = `
.c0.c0 {
  color: #554344;
  font-family: 'AIABody',Arial,'Helvetica Neue',Helvetica,sans-serif;
  font-weight: bold;
  font-size: 15px;
  line-height: 20px;
  color: #FF5500;
}

<label
  className="c0"
>
  Label
</label>
`;
